routes/cube_routes.js

```diff
@@ -1556,6 +1556,119 @@ router.get('/download/plaintext/:id', async (req, res) => {
   }
 });
 
+function shuffle(a) {
+  for (let i = a.length - 1; i > 0; i--) {
+    const j = Math.floor(Math.random() * (i + 1));
+    [a[i], a[j]] = [a[j], a[i]];
+  }
+  return a;
+}
+
+router.post('/startsealed/:id', async (req, res) => {
+  try {
+    const user = await User.findById(req.user);
+
+    if (!user) {
+      req.flash('danger', 'Please Login to build a sealed deck.');
+      return res.redirect(`cube/playtest/${req.params.id}`);
+    }
+    const params = {
```

Comment:
Couldn't you add `body('packs').toInt(), body('cards').toInt()` to the route definition? With any required validations like `{ min: 1}` added there?

Reply:
Good call